TW 258 Refresh /getting-started/creating-your-first-workspace/ (#3782)
postmanlabs/postman-docs: src/pages/docs/getting-started/creating-your-first-workspace.md

    url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"
---

> Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](https://learning.postman.com/docs/sending-requests/managing-environments/), and other components of your work in Postman.

Reviewer: ```suggestion
Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](/docs/sending-requests/managing-environments/), and other components of your work in Postman.
```

Author: Eep, thanks for catching this! 🙈

When you first open Postman, you will be in your default personal workspace. You can create more workspaces for your personal use and to collaborate with teammates. To create additional workspaces, you need to be signed into a [Postman account](/docs/getting-started/postman-account/).

To create a new workspace:

1. Select **Workspaces** in the upper left. Select **Create Workspace** to add a new workspace. You can also search for existing workspaces using the search bar or select one from the dropdown list.

    <img alt="Change workspace or create new" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.1.jpg" width="300px"/>

1. Enter a **Name** and optional **Summary** for your workspace.
1. Choose the workspace's visibility:
    * A **Personal** workspace is only visible to you.
    * A **Private** workspace is only visible to you and to any team members you invite to join it.

>     * A **Team** workspace allows you to invite collaborators.

Reviewer 2: ```suggestion
    * A **Team** workspace is visible to everyone on your team.
```

Author: Good call! Updated.
